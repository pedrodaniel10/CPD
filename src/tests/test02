1 3 1000000 20