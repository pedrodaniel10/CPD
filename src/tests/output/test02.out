0.36 0.93
0.50 0.50
